src/bindgen/cpp/cookbook.cr

```diff
@@ -100,27 +100,27 @@ module Bindgen
         "new #{class_name}"
       end
 
-      def value_to_pointer(type : String) : String
```

Comment:
I changed all these because they need to match the abstract definitions above, which are all `String?`

Reply:
Aah of course, I thought this could have been the case but didn't actually check.